core/src/main/scala/io/finch/syntax/Mapper.scala

```diff
@@ -1,7 +1,8 @@
 package io.finch.syntax
 
+import _root_.scala.annotation.implicitNotFound
```

Comment:
Do we need `_root_`?

Reply:
There is a `scala` package in a same scope, so yeah

Reply:
Ugh. Right.